fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/SplitAggregateRule.java

```diff
@@ -377,11 +377,20 @@ private Map<ColumnRefOperator, CallOperator> createNormalAgg(AggType aggType,
             Type intermediateType = getIntermediateType(aggregation);
             // For merge agg function, we need to replace the agg input args to the update agg function result
             if (aggType.isGlobal()) {
+                List<ScalarOperator> arguments =
+                        Lists.newArrayList(new ColumnRefOperator(column.getId(), intermediateType, column.getName(),
+                                column.isNullable()));
+                // For Multi args aggregation functions, if they have const args,
+                // We should also pass the const args to merge phase aggregator for performance.
+                // For example, for intersect_count(user_id, dt, '20191111'),
+                // We should pass '20191111' to update and merge phase aggregator in BE both.
```

Comment:
Why is there a performance boost?

Reply:
If we don't pass the const value by function args. we must pass them by Serde like currently intersect_count function does.